Nomster/RESTLoader.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  RESTLoader.swift
+//  Nomster
+//
+//  Created by Klaus Meyer on 06.03.15.
+//  Copyright (c) 2015 Tobias Scholze. All rights reserved.
+//
+
+import Foundation
+
+class RESTLoader {
+
+    let API_BASE = "http://localhost:3000"
```

Comment:
:+1: I was thinking about how we could make this configurable. But hey - I'm a swift / iOS noob.